### PR TITLE
Bfdr updates

### PR DESCRIPTION
**Pull request by rpassas**
### Summary
- change log added
- integrated additional updates from BFDR STU1.1 version
- moved some profiles/instances between BFDR and VRCL per harmonization plan
- made minor corrections

### CODE REVIEW

File: input/fsh/BFDR/profilesMigratingFromVRCL/ConditionPrepregnancyDiabetes_new.fsh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Profile: ConditionPrepregnancyDiabetesNew
+Parent: USCoreConditionProblemsHealthConcernsProfile
+Title: "Condition - Prepregnancy Diabetes Vital Records"
+Description: "This Condition profile indicates that the mother had a pregnancy risk factor of prepregnancy diabetes."
+* ^meta.versionId = "2"
```
Comment (ptran265) [inline input/fsh/BFDR/profilesMigratingFromVRCL/ConditionPrepregnancyDiabetes_new.fsh:5]: remove meta info on this profile (and others - eclampsia hypertension, gestational diabetes, prepregnancy diabetes/hypertension, etc)

File: input/fsh/BFDR/profiles/ObservationMotherReceivedWICFood.fsh
Comment (saulakravitz) [inline input/fsh/BFDR/profiles/ObservationMotherReceivedWICFood.fsh:1]: Are you sure this swap is right?
subject = mother and focus = child/fetus?

File: input/fsh/BFDR/profilesMigratingFromVRCL/ConditionEclampsiaHypertension_new.fsh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* ^meta.source = "#tAYxo417PTp2yn2p"
+* ^version = "1.0.0"
+* ^experimental = false
+* ^publisher = "HL7 Public Health Working Group"
+* ^contact.name = "HL7 Public Health Working Group"
+* ^contact.telecom[0].system = #url
+* ^contact.telecom[=].value = "http://www.hl7.org/Special/committees/pher/index.cfm"
+* ^contact.telecom[+].system = #email
+* ^contact.telecom[=].value = "pher@lists.HL7.org"
+* ^jurisdiction = urn:iso:std:iso:3166#US "United States of America"
```
Comment (saulakravitz) [inline input/fsh/BFDR/profilesMigratingFromVRCL/ConditionEclampsiaHypertension_new.fsh:16]: I think Tricia said this, but you can delete all of this stuff...Sushi adds it for free.

**Review by saulakravitz — APPROVED** [review]
Please respond to my/Tricia's two comments, and then merge.

File: input/fsh/BFDR/profiles/ObservationMotherReceivedWICFood.fsh
Comment (rpassas) [reply input/fsh/BFDR/profiles/ObservationMotherReceivedWICFood.fsh:1]: Yes:
[STU1.0](http://hl7.org/fhir/us/bfdr/StructureDefinition-Observation-mother-received-wic-food.html)
[STU1.1](http://build.fhir.org/ig/HL7/fhir-bfdr/StructureDefinition-Observation-mother-received-wic-food.html)

**Pull request merged**